### DISCUSSION TITLE
iOS PhotoPrism App fails to upload iPhone 16 Pro

### DISCUSSION THREAD

**Discussion by realindiahotel** [photoprism/photoprism#4649] [Q&A] [answered]
I don't know if this is the right place because I don't see an iOS app repository, but using the latest version of the PhotoPrism iOS app on iPhone 16 Pro (iOS 18.1.1) if I select multiple images to upload, it uploads 1 and then has some error in the server logs about jpg not heif or something, and then it doesn't upload the remainder of the photos. The app deletes them though, so you think you uploaded the photos, the app deletes them, but the reality is only 1 photo is uploaded.

**Comment by graciousgrey** [comment]
Thank you for your feedback! 

Which app do you use - one of [those](https://docs.photoprism.app/developer-guide/native-apps/) or our [PWA](https://docs.photoprism.app/user-guide/pwa/)?

**Comment by graciousgrey** [comment]
Is it possible that all of the files you uploaded except 1 do have a wrong extension? You can find all errors related to the import by navigating to /library/errors and search for import.

If you upload files with a wrong extension they will be added to the originals folder but cannot be indexed. If you correct the file extension of those files within the originals folder and reindex, the files should appear in your library.

**Reply by realindiahotel** [reply]
Hi, I don’t see the app on the list which made me look again and perhaps this is not an official Photo Prisom app?

![IMG_0430](https://github.com/user-attachments/assets/f54dce1c-4867-4490-bd34-6a3079a4bacc)

I am not changing the extensions so it is just whatever the iPhone is making them. But if this app is not official maybe this app is the problem and I need to contact the app developer?

**Comment by realindiahotel** [comment, marked as answer]
The photos are uploading fine from the PWA, I see now this is a third party app and the issue likely lies with the way this app is handling the files, might be changing the extensions as you suggest. I will use the PWA to upload, thanks for your help.
